core/unit_test/TestViewMapping_a.hpp

```diff
@@ -647,7 +647,7 @@ void test_view_mapping() {
 
   //----------------------------------------
 
-  {
+  if (std::is_same_v<typename Space::memory_space, Kokkos::HostSpace>) {
```

Comment:
Do we really need to satisfy this condition for all of the `static_assert`?

Reply:
No, not for the `static_asserts` but for the runtime tests.

Reply:
So why did you put the `if` around everything?

Reply:
To keep the diff minimal but I can reorganize if you prefer.

Reply:
Yes, I think it would be clearer.